elastic/kibana
# [SIEM] - Fix Jest test errors and warnings

## Summary

Resolves or mutes all warnings/errors we got when running the SIEM jest tests. This includes:

### extend error
<img width="835" alt="Screen Shot 2019-07-22 at 3 24 38 PM" src="https://user-images.githubusercontent.com/6935300/61666519-eba35d00-ac94-11e9-98c6-858a5a027f14.png">
Changing `SuggestionItemField.extend` to styled(SuggestionItemField)` fixed these _everywhere_. It's crazy, I was debugging on the breadcrumbs test which just tests a function that returns an array of objects with string values... not React code at all. Yet even that test was getting this styled-components error

### columns error
<img width="819" alt="Screen Shot 2019-07-22 at 3 24 50 PM" src="https://user-images.githubusercontent.com/6935300/61666527-f52cc500-ac94-11e9-99b1-bfaef526337c.png">
We were getting this error in a few spots. this is because we were supplying the [`columns` prop on `EuiBasicTable`](https://elastic.github.io/eui/#/display/tables) with `field` but not `name`, a required prop. Added name in. Also, `align="center"` is not valid on columns. `center` works as an arg but causes the error. Therefore, I changed us to be `right` to be compliant and [submitted an issue to Eui](https://github.com/elastic/eui/issues/2153) to update

### konva error
<img width="810" alt="Screen Shot 2019-07-22 at 3 18 00 PM" src="https://user-images.githubusercontent.com/6935300/61666686-64a2b480-ac95-11e9-962d-b387d875d60f.png">
It seems like by mocking these files, we created a new Konva instance? Removing them updates the screenshots but the tests still pass. @angorayc could you please confirm that the change in `x-pack/legacy/plugins/siem/public/components/stat_items/index.test.tsx` makes sense?

### RISON error
<img width="790" alt="Screen Shot 2019-07-22 at 3 32 19 PM" src="https://user-images.githubusercontent.com/6935300/61666933-eeeb1880-ac95-11e9-89be-887855b02b57.png">
Suppress warnings about invalid RISON as this is what we are testing in these files

## To test:

check out the tables with updated columns, and run the jest tests to see how clean and shiny they run now :)


### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
